R/simulateConsumerResource.R

```diff
@@ -135,16 +117,14 @@
 #' # make sure that for non-consumables resources for each species,
 #' # the priority is 0 (smaller than any given priority)
 #' ExampleTrophicPriority <- (ExampleE > 0) * ExampleTrophicPriority
-#' ExampleCR <- simulateConsumerResource(
+#' tse_3 <- simulateConsumerResource(
 #'     n_species = n_species,
 #'     n_resources = n_resources,
 #'     E = ExampleE,
 #'     trophic_priority = ExampleTrophicPriority,
 #'     t_end = 2000
 #' )
 #'
```

Comment:
Some of the examples are moved to vignette, however I left the ones showing how to use the function with different parameters are still in the examples section of the functions.

Reply:
That's a good solution.

Have you tested that the examples go through; using `devtools::run_examples()`?

Reply:
Yes, devtools::run_examples() doesn't give any error.